traitsui/qt4/ui_panel.py

```diff
@@ -257,7 +257,6 @@ def panel(ui):
         panel = QtGui.QTabWidget()
         # Identify ourselves as being a Tabbed group so we can later
         # distinguish this from other QTabWidgets.
-        panel.setProperty("traits_tabbed_group", True)
```

Comment:
That's fine, if it needs to be removed anyway, I just wanted to make sure it wasn't being done in error.

Could you remove the comments above the lines, since I think they no longer apply as well.